api/v1alpha1/l2vni_types.go

```diff
@@ -65,8 +65,8 @@ type HostMaster struct {
 	// +optional
 	Name string `json:"name,omitempty"`
 
-	// Type of the host interface. Currently only "bridge" is supported.
-	// +kubebuilder:validation:Enum=bridge
+	// Type of the host interface. Supports linux bridge or OVS bridge.
+	// +kubebuilder:validation:Enum=bridge;ovs-bridge
```

Comment:
how about renaming bridge to "linux-bridge"?

Reply:
I would rather do that in a follow-up. This PR is already huge, and having to deal w/ the original naming choices would only bloat this further. 

I'll comply if you want me to though.